MUDscape/driver.py

```diff
@@ -73,6 +82,22 @@ def send(self, message='', prompt=True):
 
         self.request.sendall(message.format(**Fore.__dict__).encode('utf-8'))
 
+    def bufferInput(self):
+        data = bytes()
+        newLine = "b\'\\r\\n\'"
+        backspace = "b\'\\x08\'"
+        delete = "b\'\\x7f\'"
+        #print(newLine, backspace, delete)
+
+        while True:
+            nextChar = self.request.recv(1024)
```

Comment:
If we're reading character by character I think this can be changed to:

```Python
nextChar = self.request.recv(1)
```

This code might also break when a terminal *isn't* in character mode.